oat-sa/extension-tao-itemqti-pci
# Fix/TR-4575/Revert a fix breaking compliance with the IMS standard

Related to: https://oat-sa.atlassian.net/browse/TR-4575

Revert a fix that is breaking compliance with the IMS standard.

The problem is the following:
- MathJax is not scoped, it registers itself to the global context. Therefore, only one version of it can be loaded at a time. As a result, a PCI must not include such a library as it may damage the context. If the host, or any other component, relies on a particular version and the PCI introduces another, it will conflict.
- A PCI must not request anything from the host apart from the registry hook. The host is not supposed to offer any loading capability to the PCI, nor it is supposed to offer a particular library to it.

Since the PCI cannot embed MathJax, the approach brought by the reverted fix was to request it from the host using lazy loading. This breaks the contract with the IMS standard.

**Note:** This pull request only reverts the previous fix. Therefore, it reintroduces the previous issue. Another pull request will come later to add a proper implementation. The idea behind this is to quickly fix the releases for avoiding to have broken deliveries published.

How to test:
- MathJax needs to be installed first
- create/edit an item
- insert a MathEntry PCI
- insert a Maths expression in the prompt
- save the item
- check the preview: the Maths expression should not render nor block it.
- create/edit a test and add the item to it
- publish the test, then take it: the Maths expression should not render nor block the delivery.
